nixos/modules/services/networking/hostapd.nix

```diff
@@ -62,97 +81,120 @@ in
         '';
       };
 
-      interface = mkOption {
-        default = "";
-        example = "wlp2s0";
+      APs = mkOption {
```

Comment:
I'm not sure if I'm so happy with the `APs = []` part - it's quite unintuitive compared to how the rest of the module system looks like, and `hostapd` too only knows about bssids and interfaces, not `APs`.

Can we make this similar to `networking.supplicant`? Having the description in `services.hostapd`, and a `services.hostapd.<(interface_)name>.*`, like `networking.supplicant.<name>.*`?

From my understanding, there can only be one `hostapd` process per physical interface, and configuring multiple bssids would be part of the per-interface config file, which would need to duplicated for dual/multi-band setups.